src/main/resources/logback.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
```

Comment:
what does this do?

Reply:
https://stackoverflow.com/questions/3257154/how-to-prevent-logback-from-outputting-its-own-status-at-the-start-of-every-log